# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../sqwebmail/cimap-sharedfiles.diff
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- sqwebmail-5.1.1/Makefile.am.orig	2006-03-04 15:17:58.000000000 +0100
+++ sqwebmail-5.1.1/Makefile.am	2006-04-25 13:09:38.000000000 +0200
@@ -28,8 +28,7 @@
 	pgpkeys.txt
 
 scriptbindir=$(sbindir)
-scriptbin_SCRIPTS= \
-	sharedindexinstall sharedindexsplit
+scriptbin_SCRIPTS=
 
 pkglibPROGRAMS=maildirmake deliverquota reformime makemime pcpd
 
@@ -43,10 +42,10 @@
 	sharedindexinstall sharedindexsplit
 
 pkgman8dir=@mandir@/man8
-pkgman8_DATA=deliverquota.8
+pkgman8_DATA=
 
 pkgman1dir=@mandir@/man1
-pkgman1_DATA=maildirmake.1
+pkgman1_DATA=
 
 sqwebmaild.dist: sqwebmail/sqwebmaild.dist
 	cp sqwebmail/sqwebmaild.dist .
